tests/test_backends.py

```diff
@@ -22,7 +22,7 @@ def create_safetensors_model_files(safetensors_model_path: pathlib.Path, valid:
     test_json_dict = {"a": 1, "b": 2}
     json_object = json.dumps(test_json_dict, indent=4)
 
-    for file in ["config.json", "tokenizer.json", "tokenizer_config.json"]:
+    for file in ["tokenizer.json", "tokenizer_config.json"]:
```

Comment:
Why did we remove `config.json` here? It still see it in `requires_files` in backends.py and it's still getting written in the test below.

Reply:
@alimaredia he just moved it a few lines down
what's happening here is that the test needs to verify both "valid" and "invalid" safetensors models. The way we're mocking an invalid model is for it to be missing the "config.json" file. so in this PR the config.json file only gets created for the "valid model" test case and gets skipped if valid is false

Reply:
because that's the "is or isn't valid" test that this mock implements. If a "valid" model is written it'll include `config.json`, if an invalid model is written, it'll omit that file.